examples/stackdriver_agent_roles/CLOUDSHELL_TUTORIAL.md

```diff
@@ -0,0 +1,21 @@
+# Stackdriver Agent Roles for Service Account
+
+### Configuration
+
+### Set your Project ID
+
+In the Shell below, run `gcloud config set <PROJECT_ID>`, where `<PROJECT_ID>` is the GCP project in which you wish to grant roles to the service account.
+
+### Initialize Terraform
+
+In the Shell, run `terraform init` to initialize `terraform` and download necessary support files.
+
+## Applying Changes
```

Comment:
nit: let's keep all instructions on a single page. (no need for Applying Changes to be separate from Configuration.